router/config.go

```diff
@@ -204,3 +204,16 @@ func (c *Config) LoadString(json_config string) error {
 	}
 	return c.verify()
 }
+
+type ConfigServiceConfig struct {
+	Port            string
+}
+
+func GetConfigServiceUrl() string {
+	config := ConfigServiceConfig{}
```

Comment:
Just for future reference, you can always use a map[string]interface{} instead of a struct when you work with json. Then the marshal will decode everything into map[string]interface{}. You can then just get the port using config["port"].(string) get the port as a string